src/compas_fab/ghpython/components/icons.py

```diff
@@ -0,0 +1,7 @@
+import System
+
+o = "iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAIlSURBVEhL7VM7aFRREN0mYqWIYLHkztzPW5UVQVw/+CNoYZNGC1EsRfxWFoIKQgqxUBDRThS0kbSJJqKCjSgIKdIFf6AWUQSttLCKZ+47q9n1hY1dihwYHnPmP3debRELH2vr9ZXJ+bGkfqa36Jvo4iaG9gYCds8Kvh5Uj0BOJpHTbSlUTxkP+wX4TZtvVD3HFHMjpeTMOTj/jNS8wEIzhYQDpKqBLkaCk19U/wtB/CMU+Vl4v5fUX6xxri4iKyw5ujmfnFuHYrei+EuQy+CuJAlHY4zLGfIPkHjApkji75MqgXW8LA2yEd/vQWQQ+72RuQqxogztAIqvos8EKSRX/y4Hqe43nQkuZmMXms3mEtiulf7+Nuk/sEtirtHsjI7vGmHroY8VfAFummolsK59THSCVAbO+o7xyfuDNZzZmezU1W0IYQuDr5KqBNY4jEIPqJo+yHxnMwHj81ytPxWZmIXo/XGz4R3e2upCf1iPaVv4bm54v9V0NLIaK9kZRfbA9yH97zFF3vWEkfYwpDqAhNth/2Y+VWLXVl5cqXevy+79Jg2HSFXC3gdd7sDE26x767ohEhuqG2y6qg1kFCJNPMoUdveJ1LxRFMUyxH3FvR8jVQ37mfKIIq9I9YR1XE7uf6jqUtJzI7iwq73H5GQSXzyYPoE8bQtW9BjTjsP+gQ29timYojdarVYfEg3hH3iPRJ9zok75CPkCmcSlHGbYIhY0arXfMz6/PVGPpY0AAAAASUVORK5CYII="
```

Comment:
Since we have a build step, it would probably make sense to have these icon files as files and create the base64 encoded string at compile-time.